api/tests/unit/domain/models/CertificationIssueReport_test.js

```diff
@@ -51,6 +53,81 @@ describe('Unit | Domain | Models | CertificationIssueReport', function () {
         });
       });
     });
+    context('CATEGORY: NON_BLOCKING_TECHNICAL_ISSUE', function () {
+      let certificationIssueReportDTO;
+
+      beforeEach(function () {
+        certificationIssueReportDTO = {
+          certificationCourseId: 123,
+          category: CertificationIssueReportCategories.NON_BLOCKING_TECHNICAL_ISSUE,
+          description: 'Une description obligatoire',
+        };
+      });
+
+      it('should create a NON_BLOCKING_TECHNICAL_ISSUE CertificationIssueReport', function () {
+        expect(CertificationIssueReport.create(certificationIssueReportDTO)).to.be.an.instanceOf(
+          CertificationIssueReport
+        );
+      });
+
+      // eslint-disable-next-line mocha/no-setup-in-describe
+      [MISSING_VALUE, EMPTY_VALUE, UNDEFINED_VALUE, WHITESPACES_VALUE].forEach((emptyValue) => {
+        it(`should throw an InvalidCertificationIssueReportForSaving when description is of value ${emptyValue}`, function () {
+          // when
+          expect(() =>
+            CertificationIssueReport.create({ ...certificationIssueReportDTO, description: emptyValue })
+          ).to.throw(InvalidCertificationIssueReportForSaving);
+        });
+      });
+
+      // eslint-disable-next-line mocha/no-setup-in-describe
+      [MISSING_VALUE, EMPTY_VALUE, UNDEFINED_VALUE].forEach((emptyValue) => {
+        it(`should create a NON_BLOCKING_TECHNICAL_ISSUE CertificationIssueReport when subcategory is empty with value ${emptyValue}`, function () {
+          // when
+          expect(
+            CertificationIssueReport.create({ ...certificationIssueReportDTO, subcategory: emptyValue })
+          ).to.be.an.instanceOf(CertificationIssueReport);
+        });
+      });
+    });
+
+    context('CATEGORY: NON_BLOCKING_CANDIDATE_ISSUE', function () {
+      let certificationIssueReportDTO;
+
+      beforeEach(function () {
+        certificationIssueReportDTO = {
+          certificationCourseId: 123,
+          category: CertificationIssueReportCategories.NON_BLOCKING_CANDIDATE_ISSUE,
+          description: 'Une description obligatoire',
+        };
+      });
+
+      it('should create a NON_BLOCKING_CANDIDATE_ISSUE CertificationIssueReport', function () {
+        expect(CertificationIssueReport.create(certificationIssueReportDTO)).to.be.an.instanceOf(
+          CertificationIssueReport
+        );
+      });
+
+      // eslint-disable-next-line mocha/no-setup-in-describe
+      [MISSING_VALUE, EMPTY_VALUE, UNDEFINED_VALUE, WHITESPACES_VALUE].forEach((emptyValue) => {
+        it(`should throw an InvalidCertificationIssueReportForSaving when description is of value ${emptyValue}`, function () {
```

Comment:
Mettre des "" pour bien voir l'affichage de EMPTY_VALUE
```suggestion
        it(`should throw an InvalidCertificationIssueReportForSaving when description is of value "${emptyValue}"`, function () {
```